docs/en/cloud/manage/scaling.md

```diff
@@ -6,18 +6,34 @@ slug: /en/manage/scaling
 
 # Automatic Scaling
 
-ClickHouse Cloud provides auto-scaling of your services. The scaling of ClickHouse Cloud Production services can be adjusted by organization members with the **Admin** role on the service **Settings** page.  Development services are always set to auto-scale and pause when idle.
+ClickHouse Cloud provides autoscaling for your services. The scaling of ClickHouse Cloud Production services can be adjusted by organization members with the **Admin** role on the service **Settings** page.
 
-![Settings page](images/AutoScaling.png)
+:::note
+Development services are always set to autoscale and will automatically pause when idle. Autoscaling settings for Development services cannot be changed.
+:::
+
+<img alt="Scaling settings page" style={{width: '450px', marginLeft: 0}} src={require('./images/AutoScaling.png').default} />
+
+## Adjusting total memory for your services (vertical scaling)
+
+Depending on your queries and use case, your services may require more or less memory.
 
-## Adjust scaling
+In the settings page, you can set the minimum and maximum **Total memory**. The compute allocated to your service scales linearly with its allocated memory.
 
-In Advanced scaling you can set the minimum and maximum **Total memory** and choose whether or not to allow automatic pausing of your service when it is inactive.
+Each node in your service will be allocated the same memory and CPU resources. If your service needs nodes with varying memory or CPU configurations, please contact support@clickhouse.com.
```

Comment:
Use replica everywhere instead of node.
Take out the second line in this paragraph unless we plan to support heterogeneous clusters which is not the case AFAIK.